awx/ui_next/src/components/PromptDetail/PromptDetail.jsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import { shape } from 'prop-types';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import { Chip, ChipGroup } from '@patternfly/react-core';
+import { VariablesDetail } from '@components/CodeMirrorInput';
+import { DetailList, Detail } from '@components/DetailList';
+import styled from 'styled-components';
+
+const PromptHeader = styled.h2`
+  font-weight: bold;
+`;
+
+function hasPromptData(launchData) {
+  return (
+    launchData.ask_credential_on_launch ||
+    launchData.ask_diff_mode_on_launch ||
+    launchData.ask_inventory_on_launch ||
+    launchData.ask_job_type_on_launch ||
+    launchData.ask_limit_on_launch ||
+    launchData.ask_scm_branch_on_launch ||
+    launchData.ask_skip_tags_on_launch ||
+    launchData.ask_tags_on_launch ||
+    launchData.ask_variables_on_launch ||
+    launchData.ask_verbosity_on_launch
+  );
+}
+
+function PromptDetail({ i18n, resource, launchConfig = {} }) {
+  const { defaults = {} } = launchConfig;
+  const VERBOSITY = {
```

Comment:
I think I'm going to create a small issue for centralizing the verbosity options.  It looks like we have similar logic in JobDetail.jsx, JobTemplateDetail.jsx, and JobTemplateForm.jsx.  It's certainly possible that it'll pop up in several other components in the future.

Reply:
https://github.com/ansible/awx/issues/6501

Reply:
Perfect 👍